settings.py

```diff
@@ -837,7 +837,7 @@
 # List of metrics providers to use. Available providers:
 # * 'corehq.util.metrics.datadog.DatadogMetrics'
 # * 'corehq.util.metrics.prometheus.PrometheusMetrics'
-METRICS_PROVIDERS = []
+METRICS_PROVIDERS = ['corehq.util.metrics.prometheus.PrometheusMetrics']
```

Comment:
Could you alternatively put this in the docker test localsettings? Not sure exactly what this does, but if it makes tests run while enforcing some compatibility assumptions with the metrics providers listed here, perhaps we could have test localsettings include both `PrometheusMetrics` and `DatadogMetrics`

Reply:
1bd0b7efd30 + a few to follow